modules/assim.sequential/R/Multi_Site_Constructors.R

```diff
@@ -6,6 +6,9 @@
 ##' @param var.names vector names of state variable names.
 ##' @param X a matrix of state variables. In this matrix rows represent ensembles, while columns show the variables for different sites.
 ##' @param localization.FUN This is the function that performs the localization of the Pf matrix and it returns a localized matrix with the same dimensions.
+##' @param t not used
+##' @param blocked.dis passed to `localization.FUN`
+##' @param ... passed to `localization.FUN`
```

Comment:
@DongchenZ Come to think of it, can you please check all the places in assim.sequential where I added missing `@param` lines? You know this package better than me.

Reply:
They look good to me.

Reply:
Thank you for checking!